sound/soc/sof/sof-priv.h

```diff
@@ -167,6 +167,31 @@ struct snd_sof_dsp_ops {
 	snd_pcm_uframes_t (*pcm_pointer)(struct snd_sof_dev *sdev,
 					 struct snd_pcm_substream *substream); /* optional */
 
+	/*
+	 * Except for probe_pointer,
+	 * all probe ops mandatory with DEBUG_PROBES enabled
+	 */
```

Comment:
where do we check that these ops are defined if they mandatory if DEBUG_PROBES is enabled? Can we please add the check in debug init maybe?

Reply:
Huh, some stuff I guess I'm not yet aware of, due to my _long_ experience with SOF stack. Some guidance, perhaps @ranj063 ?

Reply:
@crojewsk I am wondering where it makes sense to add these. You add the debugfs entries in snd_sof_dbg_init(). Maybe we can add the check to ensure these are set in sof_probe_continue() before invoking dbg_init() and only creating the debugfs entries if they are?